### ISSUE TITLE
Swagger UI hosted on empty path does not work anymore

### ISSUE THREAD

**Issue by LinusPhoenix** [nestjs/swagger#2006]
### Did you read the migration guide?

- [X] I have read the whole migration guide

### Is there an existing issue that is already proposing this?

- [X] I have searched the existing issues

### Versions

5.2.0 -> 6.0.4

### Describe the regression

Before 6.0, using `swagger-ui-express`, it was possible to host the swagger UI on the base url of the NestJS service. Now, the swagger UI doesn't load, rendering a white page, because requesting `swagger-ui-init.js` results in a 404.

It works if I pass another path to `SwaggerModule.setup()` (e.g., `"api"`). I've already tried passing `"/"`, but that does not work either.

### Minimum reproduction code

```ts
const config = new DocumentBuilder()
        .setTitle("foo")
        .setDescription("bar")
        .setVersion("1.0")
        .build();
    const document = SwaggerModule.createDocument(app, config);
    SwaggerModule.setup("", app, document);
```

### Expected behavior

Visiting `127.0.0.1:3000` displays the Swagger UI.

### Other

![image](https://user-images.githubusercontent.com/5871034/181477277-94fcbd07-8e97-4132-bf42-3048e1cabfe6.png)

**Comment by kamilmysliwiec** [comment]
cc @flamewow 🤔

**Comment by jarpoole** [comment]
We are also experiencing this issue since upgrading to nestjs v9. If documentation is hosted on the root url only a white page loads with a 404 occurring when `swagger-ui-init.js` is retrieved.

**Comment by LinusPhoenix** [comment]
I was just investigating how the paths are handled internally when you pass `""` or `"/"` as the path into `SwaggerModule` and when I came here to type what I found I saw you already fixed it a couple minutes ago. Thanks for the quick response!

**Comment by kamilmysliwiec** [comment]
Fixed in 6.0.5
